tests/fixtures/cluster/utilities.py

```
@@ -0,0 +1,156 @@
+        patches={
+            identity_provider_config: {
+                "metadata": {"name": identity_provider_config.name},
+                "spec": identity_providers_spec,
+            }
+        }
+    )
+    r_editor.update()
+
+
+@pytest.fixture(scope="session")
+def workers_utility_pods(admin_client, workers, utility_daemonset, installing_cnv):
```

Ruff (0.16.1)

[warning] Unused function argument: `utility_daemonset`

(ARG001)